### PR TITLE
[fixed]A write-error in contributing.md

### PR DESCRIPTION
**Pull request by kk580kk**
there got error like 
```
➜  incubator-superset git:(master) virtualenv -p python3 venv .    
Already using interpreter /usr/bin/python3
There must be only one argument: DEST_DIR (you gave venv .)
Usage: virtualenv.py [OPTIONS] DEST_DIR

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  -v, --verbose         Increase verbosity.
  -q, --quiet           Decrease verbosity.
  -p PYTHON_EXE, --python=PYTHON_EXE
                        The Python interpreter to use, e.g.,
                        --python=python3.5 will use the python3.5 interpreter
                        to create the new environment.  The default is the
                        interpreter that virtualenv was installed with
                        (/usr/bin/python3)
  --clear               Clear out the non-root install and start from scratch.
  --no-site-packages    DEPRECATED. Retained only for backward compatibility.
                        Not having access to global site-packages is now the
                        default behavior.
  --system-site-packages
                        Give the virtual environment access to the global
                        site-packages.
  --always-copy         Always copy files rather than symlinking.
  --relocatable         Make an EXISTING virtualenv environment relocatable.
                        This fixes up scripts and makes all .pth files
                        relative.
  --no-setuptools       Do not install setuptools in the new virtualenv.
  --no-pip              Do not install pip in the new virtualenv.
  --no-wheel            Do not install wheel in the new virtualenv.
  --extra-search-dir=DIR
                        Directory to look for setuptools/pip distributions in.
                        This option can be used multiple times.
  --download            Download preinstalled packages from PyPI.
  --no-download, --never-download
                        Do not download preinstalled packages from PyPI.
  --prompt=PROMPT       Provides an alternative prompt prefix for this
                        environment.
  --setuptools          DEPRECATED. Retained only for backward compatibility.
                        This option has no effect.
  --distribute          DEPRECATED. Retained only for backward compatibility.
                        This option has no effect.
  --unzip-setuptools    DEPRECATED.  Retained only for backward compatibility.
                        This option has no effect.

```
and then fixed it by change the command.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -152,8 +152,11 @@ HTML Sphinx generated should be in `docs/_build/html`. Navigate there
 and start a simple web server so we can check out the docs in a browser:
 
 ```bash
-cd docs/_build/html
+cd _build/html
+#python2
```
Comment (mistercrunch) [inline CONTRIBUTING.md]: We don't support py2.7 anymore, so might as well just use `python -m http.server 8000`. You may want to add a note about py3, or straight up add the 3 in the command as in `python3 -m http.server 8000`. But I'm not sure whether `python3` symlink is available in all/most envs...

Comment (kk580kk) [reply CONTRIBUTING.md]: >  But I'm not sure whether python3 symlink is available in all/most envs...

emmmm , i don't know how to guess symlink if available ,maybe virtualenv is a solution.

Comment (kk580kk) [reply CONTRIBUTING.md]: > We don't support py2.7 anymore, so might as well just use python -m http.server 8000. You may want to add a note about py3, or straight up add the 3 in the command as in python3 -m http.server 8000.
I change the comment  as you wish.

**Review by phoenix24 — APPROVED** [review]

**Comment by kristw** [comment]
Please rebase

**Pull request closed without merging**